about.html

```diff
@@ -0,0 +1,198 @@
+            Festival 2022. <br />
+          </p>
+          <h4>enningfestus@gmail.com</h4>
+        </div>
+      </div>
+    </section>
+    <section class="second-sec">
+      <div class="second-sec-container">
+        <div class="second-sec-heading">
+          <h2>FG Music Festival 2022 Logo</h2>
+          <div class="underline"><hr /></div>
+        </div>
+        <div class="date1">
+          <p>
+            The Logo of FG Music Festival is ddesigned by me by combining texts
+            and font-awesome icons.
+          </p>
+        </div>
+        <div class="description1">
+          <div class="desc-wrapper">
+            <div class="bx1">
+              <a href="home.html" class="logo-txt"
+                >F<i class="fa-brands fa-google"></i> MusicFest</a
+              >
+              <div class="date-wrapper"><h6>2022.11.26(Sat)</h6></div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <hr />
+
+    <section class="third-sec-container">
+      <div class="third-sec-headline">
+        <div class="title-wrapper">
+          <h2>See the past CC Global Summits</h2>
+          <div class="underline"><hr /></div>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusantium
+            facilis neque sed aliquam reprehenderit consequatur.
+          </p>
+        </div>
+      </div>
+
+      <div class="card-container">
+        <div class="card">
+          <img class="img1" src="./img/bg1.jpg" alt="" />
+          <div class="align1">
+            <div class="pos">
+              <h3>2020</h3>
+              <p>FG Music Festival 2020 in South Africa</p>
+            </div>
+          </div>
+        </div>
+        <div class="card">
+          <img class="img1" src="./img/pic1.jpg" alt="" />
+          <div class="align1">
+            <div class="pos">
+              <h3>2019</h3>
+              <p>FG Music Festival 2019 in Ethiopia</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="last-sec">
+      <h3>Partners</h3>
+      <div class="underline"><hr /></div>
+      <div class="partners-wrapper">
+        <div class="box1">
+          <i class="fa-brands fa-2x fa-soundcloud"></i>
+          <p class="first-partner">Sound Cloud</p>
+        </div>
+        <div class="box1">
+          <i class="fa-brands fa-2x fa-google"></i>
+          <p class="first-partner">Google</p>
+        </div>
+        <div class="box1">
+          <i class="fa-brands fa-2x fa-spotify"></i>
+          <p>Spotify</p>
+        </div>
+        <div class="fourth-partner box1">
+          <i class="fa-brands fa-2x fa-youtube"></i>
+          <p class="first-partner">Youtube</p>
+        </div>
+        <div class="box1">
+          <i class="fa-brands fa-3x fa-supple"></i>
+        </div>
+      </div>
+    </section>
+    <footer class="footer-sec">
```

Comment:
- [ ] Kindly ensure your footer background dark like in the Behance design below is a comparison of the designs:

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/181729189-e02eba18-1473-43d9-bb09-ea913f0e5581.png) |  ![image](https://user-images.githubusercontent.com/61408860/181729248-d6534672-b00b-4e48-bc4a-4be244fb9ec0.png)

Reply:
- [ ] Kindly check on your mobile design, the footer should be similar in design to the desktop view, below is a comparison of your footer and Behance design footer:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/181733135-74ff8053-d7d4-44ba-89d3-7f30eca90fb7.png) |  ![image](https://user-images.githubusercontent.com/61408860/181733179-69a66b93-f2ea-48ff-8b4b-1956d5993e30.png)